Index: JS lab1/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab1/index.js b/JS lab1/index.js
new file mode 100644
--- /dev/null	(date 1681041743912)
+++ b/JS lab1/index.js	(date 1681041743912)
@@ -0,0 +1,1 @@
+console.log('gg');
\ No newline at end of file
Index: JS lab1/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab1/index.html b/JS lab1/index.html
new file mode 100644
--- /dev/null	(date 1681041183688)
+++ b/JS lab1/index.html	(date 1681041183688)
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<script src="./index.js"></script>
+</body>
+</html>
\ No newline at end of file
Index: JS lab2/.idea/JS lab2.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab2/.idea/JS lab2.iml b/JS lab2/.idea/JS lab2.iml
new file mode 100644
--- /dev/null	(date 1681041310603)
+++ b/JS lab2/.idea/JS lab2.iml	(date 1681041310603)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: JS lab2/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab2/.idea/.gitignore b/JS lab2/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1681041317982)
+++ b/JS lab2/.idea/.gitignore	(date 1681041317982)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
Index: JS lab2/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab2/.idea/modules.xml b/JS lab2/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1681041310675)
+++ b/JS lab2/.idea/modules.xml	(date 1681041310675)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/JS lab2.iml" filepath="$PROJECT_DIR$/.idea/JS lab2.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: JS lab1/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab1/.idea/vcs.xml b/JS lab1/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1681041076382)
+++ b/JS lab1/.idea/vcs.xml	(date 1681041076382)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: JS lab1/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab1/.idea/modules.xml b/JS lab1/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1681041076299)
+++ b/JS lab1/.idea/modules.xml	(date 1681041076299)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/JS lab1.iml" filepath="$PROJECT_DIR$/.idea/JS lab1.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: JS lab1/.idea/JS lab1.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab1/.idea/JS lab1.iml b/JS lab1/.idea/JS lab1.iml
new file mode 100644
--- /dev/null	(date 1681041076221)
+++ b/JS lab1/.idea/JS lab1.iml	(date 1681041076221)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: JS lab1/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JS lab1/.idea/.gitignore b/JS lab1/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1681041076668)
+++ b/JS lab1/.idea/.gitignore	(date 1681041076668)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
diff --git a/JS lab1/text.txt b/JS lab1/text.txt
new file mode 100644
